src/magellan_firmware/firmware/magellan_controller/MagellanIMU.cpp

```diff
@@ -0,0 +1,63 @@
+#include "MagellanIMU.h"
+
+MagellanIMU::MagellanIMU(ros::NodeHandle &imu_handle) : 
```

Comment:
No action required, but might be nice to call this something like `IMU` or `IMUPublisher` (since it's already inside the Magellan project)